tests/e2e/support/utils.js

```diff
@@ -133,7 +133,9 @@ export function connectNodesWithFlow(flowType, startPosition, endPosition, click
           cy.wrap($endElement)
             .trigger('mousemove', { clientX: x, clientY: y, force: true })
             .wait(renderTime)
-            .click(clickPosition, { force: true });
+            .trigger('mousedown', { force: true })
+            .click(clickPosition, { force: true })
```

Comment:
Clicking while moused down? Is the `.trigger('mousedown', { force: true })` required?

Reply:
Yeah, clicking while moused down doesn't really make sense. I'll remove it and see if it changes the tests.